xtb/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, gfortran, fetchFromGitHub, cmake, makeWrapper, mkl,
+  turbomole ? null, orca ? null, cefine ? null
+}:
+let
+  # Syscalls that XTB might make.
+  binSearchPath = with lib.lists; lib.strings.makeSearchPath "bin" ([ ]
```

Comment:
is with `with lib.lists` really needed here? Wouldn't `with lib;` not more compact here?